cli/getKeychainCmd.go

```diff
@@ -0,0 +1,30 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/archethic-foundation/archethic-cli/tui/tuiutils"
+	"github.com/spf13/cobra"
+)
+
+func GetKeychainCmd() *cobra.Command {
+	getKeychainCmd := &cobra.Command{
+		Use:   "get-keychain",
+		Short: "Get keychain",
+		Run: func(cmd *cobra.Command, args []string) {
+			accessSeed, _ := cmd.Flags().GetString("access-seed")
+			keychain, err := tuiutils.AccessKeychain(endpoint.String(), accessSeed)
+			if err != nil {
+				fmt.Println(err)
+				return
+			}
+			fmt.Println("Services name | Derivation path")
```

Comment:
I think it should be better to have a well known format output like JSON for example, hence, this could be parsed and injected into another tools like `jq` to follow unix pipeline system.